platform/android/MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/testapp/activity/BaseActivityTest.java

```diff
@@ -49,8 +49,9 @@ public void beforeTest() {
       throw new RuntimeException("Could not start test for " + getActivityClass().getSimpleName() + ".\n"
         + "The ViewHierarchy doesn't contain a view with resource id = R.id.mapView or \n"
         + "the Activity doesn't contain an instance variable with a name equal to mapboxMap.\n"
-        + "You can resolve this issue be implementing the requirements above or\n add "
-        + getActivityClass().getSimpleName() + " to the excludeActivities array in `generate-test-code.js`.\n");
+        + "You can resolve this issue be adding the requirements above or\n add "
```

Comment:
typo: "You can resolve this issue _by_ adding the requirements above or\n _by adding_"